applications_tests/gls_navier_stokes_2d/gls_droplet_marangoni_effect.output

```diff
@@ -8,31 +8,31 @@ Enstrophy  : 0
 Kinetic energy : 0
 
 *****************************************************************
-Transient iteration : 1        Time : 0.001    Time step : 0.001    CFL : 0       
+Transient iteration : 1        Time : 0.001    Time step : 0.001    CFL : 0
 *****************************************************************
-Enstrophy  : 0.435612
-Kinetic energy : 1.60087e-05
+Enstrophy  : 0.435951
+Kinetic energy : 1.60093e-05
 
 *****************************************************************
-Transient iteration : 2        Time : 0.002    Time step : 0.001    CFL : 0.00809711
+Transient iteration : 2        Time : 0.002    Time step : 0.001    CFL : 0.00809705
 *****************************************************************
-Enstrophy  : 52.7355
-Kinetic energy : 0.000821466
+Enstrophy  : 24.4541
+Kinetic energy : 0.000528142
 
 *****************************************************************
-Transient iteration : 3        Time : 0.003    Time step : 0.001    CFL : 0.123131
+Transient iteration : 3        Time : 0.003    Time step : 0.001    CFL : 0.0810037
 *****************************************************************
-Enstrophy  : 195.707
-Kinetic energy : 0.00347251
+Enstrophy  : 98.8678
```

Comment:
Is such a difference to be expected?

Reply:
Good question! I think yes for two reasons:

1. For this case, I lowered the filter factor values compared to the previous version. I think this way, there is less dissipation in the fluid, so less enstrophy.
2. I changed the computation of the surface tension force F_s = sigma*k*grad_alpha. The filtered value of grad_alpha was previously used, and now it's the unfiltered value. So I think this way (again), there is less dissipation in the fluid, so less enstrophy.